code/modules/fish/fish_items.dm

```diff
@@ -0,0 +1,52 @@
+
+/obj/item/weapon/egg_scoop
+	name = "fish egg scoop"
+	desc = "A small scoop to collect fish eggs with."
+	icon = 'icons/obj/fish_items.dmi'
+	icon_state = "egg_scoop"
+	slot_flags = SLOT_BELT
+	throwforce = 0
+	w_class = 2.0
+	throw_speed = 3
+	throw_range = 7
+
+/obj/item/weapon/fish_net
+	name = "fish net"
+	desc = "A tiny net to capture fish with. It's a death sentence!"
+	icon = 'icons/obj/fish_items.dmi'
+	icon_state = "net"
+	slot_flags = SLOT_BELT
+	throwforce = 0
+	w_class = 2.0
+	throw_speed = 3
+	throw_range = 7
+
+	suicide_act(mob/user)			//"A tiny net is a death sentence: it's a net and it's tiny!" https://www.youtube.com/watch?v=FCI9Y4VGCVw
+		viewers(user) << "<span class='warning'>[user] placed the [src.name] on top of \his head, \his fingers tangled in the netting! It looks like \he's trying to commit suicide.</span>"
```

Comment:
Zuhayr had made a comment about the suicide, but must have deleted it since I no longer see it.

The act is indeed a Kung Pow reference, and the URL in the code comment is to the referenced scene(s). Watch the clip and hopefully this will make (very slightly) more sense.

Grammar stuff will be fixed though.